tests/storages_tests/test_storages.py

```diff
@@ -946,7 +946,7 @@ def test_get_best_trial(storage_mode: str, direction: StudyDirection, values: Li
         assert storage.get_best_trial(study_id).value == expected_value
 
 
-def test_get_trials_excluded_trial_ids() -> None:
+def test_get_trials_included_trial_ids() -> None:
```

Comment:
The actual error is produced by setting the large set in the `included_trial_ids` because of the specification of sqlite. So, the second call of `_get_trials` should use large set e.g. set(range(500000)).